drivers/sensor/amg88xx/Kconfig

```diff
@@ -0,0 +1,105 @@
+# Kconfig - AMG88XX infrared thermopile sensor configuration options
+
+#
+# Copyright (c) 2017 Phytec Messtechnik GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig AMG88XX
+	bool
+	prompt "AMG88XX Infrared Thermopile Sensor"
+	depends on I2C
+	default n
+	help
+	  Enable driver for AMG88XX infrared thermopile sensor.
+
+config AMG88XX_NAME
+	string
+	prompt "Driver name"
+	default "AMG88XX"
+	depends on AMG88XX
```

Comment:
Let's wrap this config and everything below it with ``if AMG88XX`` so we don't keep repeating `depends on AMG88XX`